modify test environment. (#24)
ground-x/caver-java-ext-kas: README.md

options.setCursor();
options.setFromTimestamp();
options.setToTimesatamp();
```

## Test

> To test, You must modify data in "Config.java" or use ".env" to set the data necessary for test. 

Reviewer: ```suggestion
Before testing, you need to either modify the data in "Config.java" or use ".env" to set the data required for testing.
```

> For test, You must modify data in Config.java or use .env to set the data necessary for test. 
> If you use .env, you must set variable below. 

Reviewer: ```suggestion
To test, You must modify data in "Config.java" or use ".env" to set the data necessary for test. 
If you use ".env", you must set variable below. 
```
